settings.gradle.kts

```
@@ -66,6 +66,19 @@ gradleEnterprise {
     }
 }
 
+buildCache {
+    local { isEnabled = false }
```

Do we want local cache disabled?

From https://docs.gradle.org/current/userguide/build_cache.html#sec:build_cache_configure:
> When both build caches are enabled, Gradle tries to load build outputs from the local build cache first, and then tries the remote build cache if no build outputs are found. If outputs are found in the remote cache, they are also stored in the local cache, so next time they will be found locally.

Seems like that's the behaviour we'd want, at least for non CI-builds?
```suggestion
    local { isEnabled = !isCiBuild }
```

What do you think?

---

Thank you for watching this thread! I am current at experiment 4 in the following trial now and I would be live that we always want to have local cache enabled at the end of the trial.
[Gradle Enterprise Trial Navigator for Gradle.pdf](https://github.com/detekt/detekt/files/8647982/Gradle.Enterprise.Trial.Navigator.for.Gradle.pdf)

---

(I probably will continue using this thread for testing/tweaking purposes).